src/System.Private.Xml.Linq/src/System.Private.Xml.Linq.csproj

```diff
@@ -25,7 +25,6 @@
     <Compile Include="$(CommonPath)\System\IO\StringBuilderCache.cs">
       <Link>System\StringBuilderCache.cs</Link>
     </Compile>
-    <Compile Include="Properties\InternalsVisibleTo.cs" />
```

Comment:
Why did we remove this? Do the tests not need to see the internals on this assembly anymore?